src/views/Community/Editor.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { sendCommunityData } from '../../store/actions/community'
+import {
+  Input, Row
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Input' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)